lib/AST/ASTDumper.cpp

```diff
@@ -1930,7 +1930,19 @@ class PrintExpr : public ExprVisitor<PrintExpr> {
   }
   void visitInterpolatedStringLiteralExpr(InterpolatedStringLiteralExpr *E) {
     printCommon(E, "interpolated_string_literal_expr");
-    PrintWithColorRAII(OS, LiteralValueColor) << " literal_capacity=" 
+    
+    // Print the trailing quote location
```

Comment:
If it *does* become the `EndLoc`, this should become unnecessary because the AST dump will already print the range.

Reply:
Yup!